main.go

```diff
@@ -63,6 +134,7 @@ func caduceus(arguments []string) int {
 		v = viper.New()
 
 		logger, metricsRegistry, webPA, err = server.Initialize(applicationName, arguments, f, v, Metrics, ancla.Metrics)
+		infoLogger                          = level.Info(logger)
```

Comment:
I don't have a problem with adding `infoLogger`, but there are plenty of places in `caduceus()` where we are still logging with `logger.Log(level.Key(), level.InfoValue(),...)`.  I think we should be consistent - so either log all info level logs using `infoLogger` or don't create a specific `infoLogger`.

Reply:
@joe94 can you please clarify on this?

Reply:
I agree with @kristinapathak about staying consistent. 

I would suggest updating this method so leveled log calls are of the form `level.[LEVEL](logger).Log(...)` (i.e. instead of using a variable, just do something like `level.Info(logger).Log(logging.MessageKey(), "Webhook service enabled")`). Like suggested in the docs: https://github.com/go-kit/kit/blob/master/log/level/doc.go#L14

Other examples to udpate:
1) `log.WithPrefix(logger, level.Key(), level.InfoValue()).Log("configurationFile", v.ConfigFileUsed())` **should be** 
`level.Info(logger).Log("configurationFile", v.ConfigFileUsed())`

2) `logger.Log(level.Key(), level.ErrorValue(), logging.MessageKey(), "Unable to initialize service discovery environment", logging.ErrorKey(), err)` **should be** 
`level.Error(logger).Log(logging.MessageKey(), "Unable to initialize service discovery environment", logging.ErrorKey(), err)`

Reply:
I have fixed the same in latest commit.